cheatsheets/IntelliJ_Shortcuts.rb

```diff
@@ -0,0 +1,528 @@
+cheatsheet do
+  title 'IntelliJ'
+  docset_file_name 'IntelliJ_Shortcuts'
+  keyword 'intellij'
+  source_url 'http://cheat.kapeli.com'
+  introduction 'My *Intellij* cheat sheet'  
+
+  category do
+    id 'General'
+    entry do
+      command 'CMD+0...CMD9'
+      name 'Open corresponding tool window'
+    end
+    entry do
+      command 'CMD+S'
+      name 'Save all'
+    end
+    entry do
+      command 'CMD+OPTION+Y'
+      name 'Synchronize'
+    end
+    entry do
+      command 'CMD+SHIFT+F12'
+      name 'Toggle maximizing editor'
+    end
+    entry do
+      command 'OPTION+SHIFT+I'
+      name 'Inspect current file with current profile'
+    end
+    entry do
+      command 'CMD+,'
+      name 'Open Settings dialog'
+    end
+    entry do
+      command 'CMD+;'
+      name 'Open Project Structure dialog'
+    end
+    entry do
+      command 'CMD+SHIFT+A'
+      name 'Find Action'
+    end
+  end
+
+  category do
+    id 'Debugging'
+    entry do
+      command 'F8/F7'
+      name 'Step over / into'
+    end
+    entry do
+      command 'SHIFT+F8/SHIFT+F7'
+      name 'Smart step into / Step out'
```

Comment:
Use multiple `command` instead of `/`.

So instead of `command 'SHIFT+F8/SHIFT+F7'` use this:

```
command 'SHIFT+F8'
command 'SHIFT+F7'
```